src_python/habitat_sim/utils/data/data_extractor.py

```diff
@@ -262,6 +262,7 @@ def _config_sim(self, scene_filepath, img_size):
             "color_sensor": True,  # RGBA sensor
             "semantic_sensor": True,  # Semantic sensor
             "depth_sensor": True,  # Depth sensor
+            "audio_sensor": True,  # Audio sensor
```

Comment:
This util is being run in tests compiled without audio, so creation is crashing CI.